include/openssl/ec_key.h

```diff
@@ -356,6 +328,98 @@ OPENSSL_EXPORT EC_KEY *o2i_ECPublicKey(EC_KEY **out_key, const uint8_t **inp,
 OPENSSL_EXPORT int i2o_ECPublicKey(const EC_KEY *key, unsigned char **outp);
 
 
+// EC_KEY_METHOD
+
+// ECDSA_FLAG_OPAQUE specifies that this EC_KEY_METHOD does not expose its key
+// material. This may be set if, for instance, it is wrapping some other crypto
+// API, like a platform key store. Use |EC_KEY_METHOD_set_flag| to set
+// this flag on an |EC_KEY_METHOD|.
+// This was supported in ECDSA_METHOD previously and is not set by default.
+#define ECDSA_FLAG_OPAQUE 1
+
+// EC_KEY_get_default_method returns a reference to the default
+// |EC_KEY| implementation. All |EC_KEY| objects are initialized with the
+// returned struct. This function currently calls |EC_KEY_OpenSSL| since AWS-LC
+// does not support changing/setting the default method.
+OPENSSL_EXPORT const EC_KEY_METHOD *EC_KEY_get_default_method(void);
+
+// EC_KEY_OpenSSL returns a reference to the default |EC_KEY| implementation.
+// The returned |EC_KEY_METHOD| object is statically allocated. The application
+// should not free this struct.
+//
+// This struct is also zero-initialized. This is different from OpenSSL which
+// returns function pointers to the default implementations within the
+// |EC_KEY_METHOD| struct. We do not do this to make it easier for the
+// compiler/linker to drop unused functions. The wrapper functions will select
+// the appropriate |ec_key_default_*| implementation.
```

Comment:
- I'm not sure why we decided not to initialise with our default ECDSA implementation if that was possible.
- Where is an example of the wrapper functions selecting the appropriate |ec_key_default_*| implementation?

Reply:
This was a previous change in our library but it doesn't impact the usage or behavior of this struct/API. I have updated the documentation to be clearer about what it means for a default operation to be called. 

In the following link, the wrapper function ECDSA_sign checks for the existence of a custom implementation. In the default case, the ec_key_meth->sign pointer is NULL and so the code continues to execute the rest of the sign functionality (the default implementation). Otherwise, the custom functionality is invoked. ([link](https://github.com/aws/aws-lc/blob/70c6dfd09da529810cc9efebf32dbd9c25c3868a/crypto/fipsmodule/ecdsa/ecdsa.c#L379))